### PR TITLE
fix pep8 error

### PR DESCRIPTION
**Pull request by jichenjc**

### CODE REVIEW

**Comment by jichenjc** [comment]
```
pep8 run-test-pre: PYTHONHASHSEED='84345641'
pep8 run-test: commands[0] | flake8
pep8 run-test: commands[1] | flake8 /root/python-zvm-sdk/scripts/sdkserver
_______________________________________________________________ summary _______________________________________________________________
  pep8: commands succeeded
  congratulations :)


Ran 12 tests in 0.003s

OK
_______________________________________________________________ summary _______________________________________________________________
  py27: commands succeeded
  congratulations :)

```

File: smtLayer/smtTest.py [hunk 1]
```diff
@@ -1401,8 +1401,8 @@ def runTest(smt, test):
             if 'output' in e:
                 results['response'] = e.output
             else:
-                results['response'] = ('Exception encountered: %s, ' +
-                    "details: %s" % (type(e).__name__, str(e)))
+                results['response'] = ('Exception encountered: ' +
+                    "details: %s" % str(e))
```
Comment (bjhuangr) [inline smtLayer/smtTest.py:1405]: six.text_type(e) ?

Comment (bjhuangr) [reply smtLayer/smtTest.py:1405]: for py3 support

**Comment by bjhuangr** [comment]
minor change, no functional impact

**Pull request merged**